public/app/features/alerting/unified/PanelAlertTabContent.test.tsx

```diff
@@ -302,58 +301,7 @@ describe('PanelAlertTabContent', () => {
     expect(match).toHaveLength(2);
 
     const defaults = JSON.parse(decodeURIComponent(match![1]));
-    expect(defaults).toEqual({
```

Comment:
💬 I've moved the output of this test that asserts the data structure of a default alert to a snapshot as it's easier to work with and update when we change it later (to a reduce + threshold expression).